scout/constants/clnsig.py

```diff
@@ -28,6 +29,7 @@
     "benign": 2,
     "likely_benign": 3,
     "conflicting_interpretations_of_pathogenicity": 8,
+    "conflicting_interpretations": 8,
```

Comment:
This is possibly fixing a bug we never noticed before. Some variants have `conflicting_interpretations` instead of `conflicting_interpretations_of_pathogenicity` as clnsig value and since the option was not a valid key these accessions were not saved in the variant object